ncbi_genome_download/__main__.py

```diff
@@ -9,21 +9,21 @@
 def main():
     """Build and parse command line"""
     parser = argparse.ArgumentParser()
-    parser.add_argument('domain',
-                        choices=dflt.DOMAIN.choices,
-                        default=dflt.DOMAIN.default,
-                        help='The NCBI "domain" to download (default: %(default)s)')
+    parser.add_argument('group',
```

Comment:
I can't find the README file I had for this. I guess the NCBI gets to rename their stuff if they like. For sure "taxonomic group" is a more correct term anyway. Let's just keep it for now.